### ISSUE TITLE
Context Menu Icons Are Stuck in the Dark Theme

### ISSUE THREAD

**Issue by dotjax** [numixproject/numix-gtk-theme#736]
_Note: I wasn't sure if this was Numix's issue or the Papirus team, so I'm going to share the issue with both teams._ 

Distro: Xubuntu 18.04.2 LTS

For some reason, the context menus on most themes except Arc have a weird issue. I'm using Papirus (not the dark or light theme, just the regular one provided from the ppa) and when using Numix GTK (all versions) I get this bizarre context menu problem. The issue exists in Audacious, right clicking the desktop, inside Thunar, etc.

This happens when I select the regular Papirus theme. I'm not doing any crazy tweaks, just defaults with the newest Papirus theme provided from the Git repo.

image
![image](https://user-images.githubusercontent.com/12740958/62252289-0625b680-b3b8-11e9-9558-294cd2948612.png)

**Comment by Foggalong** [comment]
Can you reproduce this issue with any other combinations of theme? (e.g. Numix & Numix, Papirus & Arc, etc)

**Comment by dotjax** [comment]
Arc does not give me the same issue. I did contact the Papirus team and they mentioned that Xfce does not support symbolic icons in the context menu, which might be the problem. Papirus and Arc work fine, Numix and Numix works great. Numix and Papirus seem to be the issue. My guess is it having to do with symbolic links/icons. I'll test other icons over the next few hours.

**Comment by dotjax** [comment]
To add to this, here is the other thread with some more insight from the Papirus dev team:

https://github.com/PapirusDevelopmentTeam/papirus-icon-theme/issues/1836

**Edit:** This happens in the Audacious menu as well.

![image](https://user-images.githubusercontent.com/12740958/62310979-53edfd80-b450-11e9-9173-5da9646f1f23.png)

**Comment by Foggalong** [comment]
That's annoying. It does sound like an issue with XFCE rather than either theme Feel free to keep testing but provisionally I'd recommend using a "light" icon theme with "dark" GTK theme or vis vera. XFCE is a little different when it comes to things like this so often some workarounds are needed unfortunately :disappointed:

**Comment by dotjax** [comment]
No worries! Thanks for being so polite and responsive. Maybe Xfce 4.14 will solve some of this issue. I'll keep investigating. I will note that using Arc (the non-dark version), the issue still occurs. I tried using Numix with the Papirus Icon Theme and it problem exists in both dark and light variants. Pic attached.

![image](https://user-images.githubusercontent.com/12740958/62311227-01611100-b451-11e9-9d2d-8349e7b2a0eb.png)

**Comment by dotjax** [comment]
Papirus Icon Theme w/Numix: 
![image](https://user-images.githubusercontent.com/12740958/62311503-9401b000-b451-11e9-8a47-088caf7bb600.png)

**Comment by Foggalong** [comment]
How does it look with Papirus dark?

**Comment by dotjax** [comment]
It's in the pic above, same issue but as you can see, now the Thunar icons are set for dark, which makes them hard to see. This happens in other apps, too.

**Comment by Foggalong** [comment]
Oh huh, that's really weird. It's as if XFCE is actively making the symbolic icons the wrong colour

**Comment by dotjax** [comment]
I know, right? Really strange. I'll keep digging. Would you like any logs or anything?

**Comment by Foggalong** [comment]
No logs spring to mind for this (since there's no "error" actually occurring), but do let me know what you turf up on this. I've not experienced this but as an XFCE user myself it's piqued my curiosity that this can happen

**Comment by dotjax** [comment]
Woops. Didn't mean to close this. Alright, I'll keep digging.

**Comment by khurshid-alam** [comment]
I can't reproduce this on xfce. Though I am using xfce & numix from git master.

**Comment by dotjax** [comment]
Try it with non-Git. Do a VM or clean install of Xubuntu 18.04. I was just using whatever came with the stock repos. I did install Numix daily and I'm still have the same problem. I may try installing the pre-release of Xfce and see if that fixes it.

**Comment by khurshid-alam** [comment]
18.04 has gtk2 and I am not touching gtk2 unless it's a serious issue. 

Numix (default) + Papirus (dark) = works
Numix (dark) + Papirus (light) = works

Right ?

**Comment by dotjax** [comment]
No, if you look at my pic, you can see the icons in Thunar are now white/light. I'll take another pic:
![image](https://user-images.githubusercontent.com/12740958/62402491-96493480-b54d-11e9-8bf3-9cb98bbb1414.png)

Another pic:
![image](https://user-images.githubusercontent.com/12740958/62402541-d4465880-b54d-11e9-8aaf-bb2484c83064.png)

**Comment by dotjax** [comment]
It's not a huge deal, though. Xfce 4.14 will be out soon enough. 

> 18.04 has gtk2 and I am not touching gtk2 unless it's a serious issue.

Yeah I don't blame you.

**Comment by dotjax** [comment]
Just came back to say that Xfce 4.14 is out. I'm unsure if this was solved yet. Is there a PPA I can use for testing?
